### PR TITLE
TicketService - Extended Functionality with Junit5 and Mockito test integration

### PR DESCRIPTION
**Pull request by Dana0914**
-Created @Test methods
-Covered with positive, negative and corner cases
-Used Assertions
-Used Mockito to mock db repository

### CODE REVIEW

File: src/test/java/kz/runtime/ticketservicespring/TicketServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.dao.DataIntegrityViolationException;
+
+import java.time.LocalDate;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.Optional;
+
+import static org.mockito.Mockito.*;
```
Comment (deolexx) [inline src/test/java/kz/runtime/ticketservicespring/TicketServiceTest.java:22]: remove this wildcard import

File: src/test/java/kz/runtime/ticketservicespring/TicketServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+@ExtendWith(MockitoExtension.class)
```
Comment (deolexx) [inline src/test/java/kz/runtime/ticketservicespring/TicketServiceTest.java:23]: And here should be an empty line

**Review by deolexx — APPROVED** [review]
